src/community/design-system-day/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Join us for Design System Day 2023
+description: A 2-day conference to collaborate and share knowledge about design systems with like-minded people.
+section: Community
+theme: Events and workshops
+layout: layout-pane.njk
+order: 11
+---
+
+[BANNER IMAGE]
+
+Dates: 10 and 11 October 2023
+Location: Online and <a href='https://dynamicearth.org.uk/plan-your-visit/getting-here/'>Dynamic Earth</a>, Edinburgh
+
+We plan to release tickets at the start of September. <a href='https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system'>Sign up to our mailing list</a> to get updates.
+
+<!--
+
+Setting the following convention:
+    /community/design-system-day/ always describes the upcoming event or the event in general
+    /community/design-system-day-[year]/ is the archive page for an event which collects the videos, slides and notes for a particular conference
+
+This means that hyperlinks to /community/design-system-day/ can always encourage ticket sales or mailing list subscriptions.
+
+-->
+
+## Two days of talks and workshops
+
+Join us for this year's edition of the GOV.UK Design System's flagship event!
+
+Design System Day 2023 is a two-day conference to collaborate and share knowledge about design systems with like-minded people, covering topics like accessibility, community and decision-making. It's a space for anyone using design systems in government to share ideas and developments, learn from colleagues, and spark conversations on related topics. 
+
+For the first time, we're running this as a hybrid conference with a mix of in-person and online talks, workshops and panel discussions. Come say hi and meet folks from across the community in Edinburgh, or chat to people in the online channels.
+
+We’re expecting that the mix of attendees will lead to some interesting discussions and make the event a great place to network. We’re curating a space for engaging discussions, learning opportunities and future collaboration.
+
+We'll also be hosting get-togethers and after-parties for people attending the Edinburgh event. We advise you to book your travel and accommodation as soon as possible as Edinburgh fills up quickly!
```

Comment:
Tricky message here, as attendees would not secure a ticket until they're released (expected) at the start of September.